code-challenges.test.js

```diff
@@ -57,3 +160,26 @@ const vowelTester3 = "challenges"
 // Expected output: 2
 
 // b) Create the function that makes the test pass.
+
+// Process
+// create a function named "vowelLocator"
+
+// create another function named "isVowel" to that will identify vowels
+
+// Use the split operator to turn the string into an array and the find index operator to locate the index where the vowel is located
+
```

Comment:
I love this approach and really cool that you are using findIndex. I would encourage you to refactor this into one function and expand on your process.